src/imitation/algorithms/preference_comparisons.py

```diff
@@ -201,24 +207,42 @@ def __init__(
                 how good a fragment is. Default is to use undiscounted
                 sums of rewards (as in the DRLHP paper).
             seed: seed for the internal RNG (only used if temperature > 0)
+            threshold: preferences are sampled from a softmax of returns.
+                To avoid overflows, we clip differences in returns that are
+                above this threshold (after multiplying with temperature).
+                This threshold is therefore in logspace. The default value
+                of 50 means that probabilities below 2e-22 are rounded to 0.
```

Comment:
From looking at the code, I think it's the opposite? That probabilities are rounded up to 2e-22, since |returns_diff| is clamped to at most threshold. Not that this makes a practical difference.

Reply:
Oh, you're completely right. Fixed